testing/libguestfs/APKBUILD

```diff
@@ -0,0 +1,114 @@
+# Contributor:
+# Maintainer: Fathi Boudra <fathi.boudra@linaro.org>
+pkgname=libguestfs
+pkgver=1.40.2
+pkgrel=0
+pkgdesc="libguestfs"
```

Comment:
Could you improve this description? It should be more then just the package name repeated.